lib/ansible/plugins/lookup/hashi_vault.py

```diff
@@ -162,7 +177,11 @@ def __init__(self, **kwargs):
                     self.client = hvac.Client(url=self.url, verify=self.verify)
                 # prefixing with auth_ to limit which methods can be accessed
                 getattr(self, 'auth_' + self.auth_method)(**kwargs)
```

Comment:
The call to `getattr` is blowing up with the below error when the `fetch_url` is imported. Prior to using `fetch_url` I was using the `requests` module and everything was working.

Here is the error that `getattr` is throwing now ...

```
'HashiVault' object has no attribute 'module'
```

Reply:
Removed support for GCP, so no an issue anymore.